### ISSUE TITLE
Canot run apt-get update

### ISSUE THREAD

**Issue by dinukasal** [jangrewe/gitlab-ci-android#37]
When running apt-get update it gives this error,

`Running with gitlab-runner 10.4.0 (857480b6)
  on docker-auto-scale (fa6cab46)
Using Docker executor with image jangrewe/gitlab-ci-android ...
Using docker image sha256:1cde072c1921adcceafd8c55edfb73144cb78841566de60ffcd43317209aa761 for predefined container...
Pulling docker image jangrewe/gitlab-ci-android ...
Using docker image jangrewe/gitlab-ci-android ID=sha256:a61c4b9005d895f4af9bb1291002ce7ac9791e70d83aefa229c897309e9a2b49 for build container...
Running on runner-fa6cab46-project-5119850-concurrent-0 via runner-fa6cab46-srm-1517637890-433e967f...
Cloning repository...
Cloning into '/builds/dinukasal/rn_passenger_app'...
Checking out 8cbb3798 as master...
Skipping Git submodules setup
$ curl -sL https://deb.nodesource.com/setup_8.x | bash

## Populating apt-get cache...

+ apt-get update
Ign:1 http://security.ubuntu.com/ubuntu zesty-security InRelease
Ign:2 http://archive.ubuntu.com/ubuntu zesty InRelease
Ign:3 http://security.ubuntu.com/ubuntu zesty-security Release
Ign:4 http://archive.ubuntu.com/ubuntu zesty-updates InRelease
Ign:5 http://security.ubuntu.com/ubuntu zesty-security/universe Sources
Ign:6 http://archive.ubuntu.com/ubuntu zesty-backports InRelease
Ign:7 http://security.ubuntu.com/ubuntu zesty-security/restricted amd64 Packages
Ign:8 http://archive.ubuntu.com/ubuntu zesty Release
Ign:9 http://security.ubuntu.com/ubuntu zesty-security/multiverse all Packages
Ign:10 http://archive.ubuntu.com/ubuntu zesty-updates Release
Ign:11 http://security.ubuntu.com/ubuntu zesty-security/restricted all Packages
Ign:12 http://archive.ubuntu.com/ubuntu zesty-backports Release
Ign:13 http://security.ubuntu.com/ubuntu zesty-security/universe all Packages
Ign:14 http://archive.ubuntu.com/ubuntu zesty/universe Sources
Ign:15 http://security.ubuntu.com/ubuntu zesty-security/universe amd64 Packages
Ign:16 http://archive.ubuntu.com/ubuntu zesty/main all Packages
Ign:17 http://security.ubuntu.com/ubuntu zesty-security/multiverse amd64 Packages
Ign:18 http://archive.ubuntu.com/ubuntu zesty/main amd64 Packages
Ign:19 http://security.ubuntu.com/ubuntu zesty-security/main amd64 Packages
Ign:20 http://archive.ubuntu.com/ubuntu zesty/restricted amd64 Packages
Ign:21 http://security.ubuntu.com/ubuntu zesty-security/main all Packages
Ign:22 http://archive.ubuntu.com/ubuntu zesty/multiverse all Packages
Ign:5 http://security.ubuntu.com/ubuntu zesty-security/universe Sources
Ign:23 http://archive.ubuntu.com/ubuntu zesty/universe amd64 Packages
Ign:7 http://security.ubuntu.com/ubuntu zesty-security/restricted amd64 Packages
Ign:24 http://archive.ubuntu.com/ubuntu zesty/universe all Packages
Ign:9 http://security.ubuntu.com/ubuntu zesty-security/multiverse all Packages
Ign:25 http://archive.ubuntu.com/ubuntu zesty/restricted all Packages
Ign:11 http://security.ubuntu.com/ubuntu zesty-security/restricted all Packages
Ign:26 http://archive.ubuntu.com/ubuntu zesty/multiverse amd64 Packages
Ign:13 http://security.ubuntu.com/ubuntu zesty-security/universe all Packages
Ign:27 http://archive.ubuntu.com/ubuntu zesty-updates/universe Sources
Ign:15 http://security.ubuntu.com/ubuntu zesty-security/universe amd64 Packages
Ign:28 http://archive.ubuntu.com/ubuntu zesty-updates/multiverse amd64 Packages
Ign:17 http://security.ubuntu.com/ubuntu zesty-security/multiverse amd64 Packages
Ign:29 http://archive.ubuntu.com/ubuntu zesty-updates/main amd64 Packages
Ign:19 http://security.ubuntu.com/ubuntu zesty-security/main amd64 Packages
Ign:30 http://archive.ubuntu.com/ubuntu zesty-updates/universe all Packages
Ign:21 http://security.ubuntu.com/ubuntu zesty-security/main all Packages
Ign:31 http://archive.ubuntu.com/ubuntu zesty-updates/multiverse all Packages
Ign:5 http://security.ubuntu.com/ubuntu zesty-security/universe Sources
Ign:32 http://archive.ubuntu.com/ubuntu zesty-updates/main all Packages
Ign:7 http://security.ubuntu.com/ubuntu zesty-security/restricted amd64 Packages
Ign:33 http://archive.ubuntu.com/ubuntu zesty-updates/restricted amd64 Packages
Ign:9 http://security.ubuntu.com/ubuntu zesty-security/multiverse all Packages
Ign:34 http://archive.ubuntu.com/ubuntu zesty-updates/universe amd64 Packages
Ign:35 http://archive.ubuntu.com/ubuntu zesty-updates/restricted all Packages
Ign:11 http://security.ubuntu.com/ubuntu zesty-security/restricted all Packages
Ign:13 http://security.ubuntu.com/ubuntu zesty-security/universe all Packages
Ign:36 http://archive.ubuntu.com/ubuntu zesty-backports/universe amd64 Packages
Ign:15 http://security.ubuntu.com/ubuntu zesty-security/universe amd64 Packages
Ign:37 http://archive.ubuntu.com/ubuntu zesty-backports/main amd64 Packages
Ign:17 http://security.ubuntu.com/ubuntu zesty-security/multiverse amd64 Packages
Ign:38 http://archive.ubuntu.com/ubuntu zesty-backports/universe all Packages
Ign:19 http://security.ubuntu.com/ubuntu zesty-security/main amd64 Packages
Ign:39 http://archive.ubuntu.com/ubuntu zesty-backports/multiverse all Packages
Ign:21 http://security.ubuntu.com/ubuntu zesty-security/main all Packages
Ign:40 http://archive.ubuntu.com/ubuntu zesty-backports/restricted all Packages
Ign:5 http://security.ubuntu.com/ubuntu zesty-security/universe Sources
Ign:41 http://archive.ubuntu.com/ubuntu zesty-backports/main all Packages
Ign:7 http://security.ubuntu.com/ubuntu zesty-security/restricted amd64 Packages
Ign:42 http://archive.ubuntu.com/ubuntu zesty-backports/restricted amd64 Packages
Ign:9 http://security.ubuntu.com/ubuntu zesty-security/multiverse all Packages
Ign:43 http://archive.ubuntu.com/ubuntu zesty-backports/multiverse amd64 Packages
Ign:11 http://security.ubuntu.com/ubuntu zesty-security/restricted all Packages
Ign:14 http://archive.ubuntu.com/ubuntu zesty/universe Sources
Ign:13 http://security.ubuntu.com/ubuntu zesty-security/universe all Packages
Ign:16 http://archive.ubuntu.com/ubuntu zesty/main all Packages
Ign:15 http://security.ubuntu.com/ubuntu zesty-security/universe amd64 Packages
Ign:18 http://archive.ubuntu.com/ubuntu zesty/main amd64 Packages
Ign:17 http://security.ubuntu.com/ubuntu zesty-security/multiverse amd64 Packages
Ign:20 http://archive.ubuntu.com/ubuntu zesty/restricted amd64 Packages
Ign:19 http://security.ubuntu.com/ubuntu zesty-security/main amd64 Packages
Ign:22 http://archive.ubuntu.com/ubuntu zesty/multiverse all Packages
Ign:21 http://security.ubuntu.com/ubuntu zesty-security/main all Packages
Ign:23 http://archive.ubuntu.com/ubuntu zesty/universe amd64 Packages
Ign:5 http://security.ubuntu.com/ubuntu zesty-security/universe Sources
Ign:24 http://archive.ubuntu.com/ubuntu zesty/universe all Packages
Ign:7 http://security.ubuntu.com/ubuntu zesty-security/restricted amd64 Packages
Ign:25 http://archive.ubuntu.com/ubuntu zesty/restricted all Packages
Ign:9 http://security.ubuntu.com/ubuntu zesty-security/multiverse all Packages
Ign:26 http://archive.ubuntu.com/ubuntu zesty/multiverse amd64 Packages
Ign:11 http://security.ubuntu.com/ubuntu zesty-security/restricted all Packages
Ign:27 http://archive.ubuntu.com/ubuntu zesty-updates/universe Sources
Ign:13 http://security.ubuntu.com/ubuntu zesty-security/universe all Packages
Ign:28 http://archive.ubuntu.com/ubuntu zesty-updates/multiverse amd64 Packages
Ign:15 http://security.ubuntu.com/ubuntu zesty-security/universe amd64 Packages
Ign:29 http://archive.ubuntu.com/ubuntu zesty-updates/main amd64 Packages
Ign:17 http://security.ubuntu.com/ubuntu zesty-security/multiverse amd64 Packages
Ign:30 http://archive.ubuntu.com/ubuntu zesty-updates/universe all Packages
Ign:19 http://security.ubuntu.com/ubuntu zesty-security/main amd64 Packages
Ign:31 http://archive.ubuntu.com/ubuntu zesty-updates/multiverse all Packages
Ign:21 http://security.ubuntu.com/ubuntu zesty-security/main all Packages
Ign:32 http://archive.ubuntu.com/ubuntu zesty-updates/main all Packages
Ign:33 http://archive.ubuntu.com/ubuntu zesty-updates/restricted amd64 Packages
Err:5 http://security.ubuntu.com/ubuntu zesty-security/universe Sources
  404  Not Found [IP: 91.189.88.149 80]
Ign:34 http://archive.ubuntu.com/ubuntu zesty-updates/universe amd64 Packages
Ign:7 http://security.ubuntu.com/ubuntu zesty-security/restricted amd64 Packages
Ign:9 http://security.ubuntu.com/ubuntu zesty-security/multiverse all Packages
Ign:35 http://archive.ubuntu.com/ubuntu zesty-updates/restricted all Packages
Ign:11 http://security.ubuntu.com/ubuntu zesty-security/restricted all Packages
Ign:36 http://archive.ubuntu.com/ubuntu zesty-backports/universe amd64 Packages
Ign:13 http://security.ubuntu.com/ubuntu zesty-security/universe all Packages
Ign:37 http://archive.ubuntu.com/ubuntu zesty-backports/main amd64 Packages
Ign:15 http://security.ubuntu.com/ubuntu zesty-security/universe amd64 Packages
Ign:38 http://archive.ubuntu.com/ubuntu zesty-backports/universe all Packages
Ign:17 http://security.ubuntu.com/ubuntu zesty-security/multiverse amd64 Packages
Ign:39 http://archive.ubuntu.com/ubuntu zesty-backports/multiverse all Packages
Ign:19 http://security.ubuntu.com/ubuntu zesty-security/main amd64 Packages
Ign:40 http://archive.ubuntu.com/ubuntu zesty-backports/restricted all Packages
Ign:21 http://security.ubuntu.com/ubuntu zesty-security/main all Packages
Ign:41 http://archive.ubuntu.com/ubuntu zesty-backports/main all Packages
Ign:42 http://archive.ubuntu.com/ubuntu zesty-backports/restricted amd64 Packages
Ign:43 http://archive.ubuntu.com/ubuntu zesty-backports/multiverse amd64 Packages
Ign:14 http://archive.ubuntu.com/ubuntu zesty/universe Sources
Ign:16 http://archive.ubuntu.com/ubuntu zesty/main all Packages
Ign:18 http://archive.ubuntu.com/ubuntu zesty/main amd64 Packages
Ign:20 http://archive.ubuntu.com/ubuntu zesty/restricted amd64 Packages
Ign:22 http://archive.ubuntu.com/ubuntu zesty/multiverse all Packages
Ign:23 http://archive.ubuntu.com/ubuntu zesty/universe amd64 Packages
Ign:24 http://archive.ubuntu.com/ubuntu zesty/universe all Packages
Ign:25 http://archive.ubuntu.com/ubuntu zesty/restricted all Packages
Ign:26 http://archive.ubuntu.com/ubuntu zesty/multiverse amd64 Packages
Ign:27 http://archive.ubuntu.com/ubuntu zesty-updates/universe Sources
Ign:28 http://archive.ubuntu.com/ubuntu zesty-updates/multiverse amd64 Packages
Ign:29 http://archive.ubuntu.com/ubuntu zesty-updates/main amd64 Packages
Ign:30 http://archive.ubuntu.com/ubuntu zesty-updates/universe all Packages
Ign:31 http://archive.ubuntu.com/ubuntu zesty-updates/multiverse all Packages
Ign:32 http://archive.ubuntu.com/ubuntu zesty-updates/main all Packages
Ign:33 http://archive.ubuntu.com/ubuntu zesty-updates/restricted amd64 Packages
Ign:34 http://archive.ubuntu.com/ubuntu zesty-updates/universe amd64 Packages
Ign:35 http://archive.ubuntu.com/ubuntu zesty-updates/restricted all Packages
Ign:36 http://archive.ubuntu.com/ubuntu zesty-backports/universe amd64 Packages
Ign:37 http://archive.ubuntu.com/ubuntu zesty-backports/main amd64 Packages
Ign:38 http://archive.ubuntu.com/ubuntu zesty-backports/universe all Packages
Ign:39 http://archive.ubuntu.com/ubuntu zesty-backports/multiverse all Packages
Ign:40 http://archive.ubuntu.com/ubuntu zesty-backports/restricted all Packages
Ign:41 http://archive.ubuntu.com/ubuntu zesty-backports/main all Packages
Ign:42 http://archive.ubuntu.com/ubuntu zesty-backports/restricted amd64 Packages
Ign:43 http://archive.ubuntu.com/ubuntu zesty-backports/multiverse amd64 Packages
Ign:14 http://archive.ubuntu.com/ubuntu zesty/universe Sources
Ign:16 http://archive.ubuntu.com/ubuntu zesty/main all Packages
Ign:18 http://archive.ubuntu.com/ubuntu zesty/main amd64 Packages
Ign:20 http://archive.ubuntu.com/ubuntu zesty/restricted amd64 Packages
Ign:22 http://archive.ubuntu.com/ubuntu zesty/multiverse all Packages
Ign:23 http://archive.ubuntu.com/ubuntu zesty/universe amd64 Packages
Ign:24 http://archive.ubuntu.com/ubuntu zesty/universe all Packages
Ign:25 http://archive.ubuntu.com/ubuntu zesty/restricted all Packages
Ign:26 http://archive.ubuntu.com/ubuntu zesty/multiverse amd64 Packages
Ign:27 http://archive.ubuntu.com/ubuntu zesty-updates/universe Sources
Ign:28 http://archive.ubuntu.com/ubuntu zesty-updates/multiverse amd64 Packages
Ign:29 http://archive.ubuntu.com/ubuntu zesty-updates/main amd64 Packages
Ign:30 http://archive.ubuntu.com/ubuntu zesty-updates/universe all Packages
Ign:31 http://archive.ubuntu.com/ubuntu zesty-updates/multiverse all Packages
Ign:32 http://archive.ubuntu.com/ubuntu zesty-updates/main all Packages
Ign:33 http://archive.ubuntu.com/ubuntu zesty-updates/restricted amd64 Packages
Ign:34 http://archive.ubuntu.com/ubuntu zesty-updates/universe amd64 Packages
Ign:35 http://archive.ubuntu.com/ubuntu zesty-updates/restricted all Packages
Ign:36 http://archive.ubuntu.com/ubuntu zesty-backports/universe amd64 Packages
Ign:37 http://archive.ubuntu.com/ubuntu zesty-backports/main amd64 Packages
Ign:38 http://archive.ubuntu.com/ubuntu zesty-backports/universe all Packages
Ign:39 http://archive.ubuntu.com/ubuntu zesty-backports/multiverse all Packages
Ign:40 http://archive.ubuntu.com/ubuntu zesty-backports/restricted all Packages
Ign:41 http://archive.ubuntu.com/ubuntu zesty-backports/main all Packages
Ign:42 http://archive.ubuntu.com/ubuntu zesty-backports/restricted amd64 Packages
Ign:43 http://archive.ubuntu.com/ubuntu zesty-backports/multiverse amd64 Packages
Ign:14 http://archive.ubuntu.com/ubuntu zesty/universe Sources
Ign:16 http://archive.ubuntu.com/ubuntu zesty/main all Packages
Ign:18 http://archive.ubuntu.com/ubuntu zesty/main amd64 Packages
Ign:20 http://archive.ubuntu.com/ubuntu zesty/restricted amd64 Packages
Ign:22 http://archive.ubuntu.com/ubuntu zesty/multiverse all Packages
Ign:23 http://archive.ubuntu.com/ubuntu zesty/universe amd64 Packages
Ign:24 http://archive.ubuntu.com/ubuntu zesty/universe all Packages
Ign:25 http://archive.ubuntu.com/ubuntu zesty/restricted all Packages
Ign:26 http://archive.ubuntu.com/ubuntu zesty/multiverse amd64 Packages
Ign:27 http://archive.ubuntu.com/ubuntu zesty-updates/universe Sources
Ign:28 http://archive.ubuntu.com/ubuntu zesty-updates/multiverse amd64 Packages
Ign:29 http://archive.ubuntu.com/ubuntu zesty-updates/main amd64 Packages
Ign:30 http://archive.ubuntu.com/ubuntu zesty-updates/universe all Packages
Ign:31 http://archive.ubuntu.com/ubuntu zesty-updates/multiverse all Packages
Ign:32 http://archive.ubuntu.com/ubuntu zesty-updates/main all Packages
Ign:33 http://archive.ubuntu.com/ubuntu zesty-updates/restricted amd64 Packages
Ign:34 http://archive.ubuntu.com/ubuntu zesty-updates/universe amd64 Packages
Ign:35 http://archive.ubuntu.com/ubuntu zesty-updates/restricted all Packages
Ign:36 http://archive.ubuntu.com/ubuntu zesty-backports/universe amd64 Packages
Ign:37 http://archive.ubuntu.com/ubuntu zesty-backports/main amd64 Packages
Ign:38 http://archive.ubuntu.com/ubuntu zesty-backports/universe all Packages
Ign:39 http://archive.ubuntu.com/ubuntu zesty-backports/multiverse all Packages
Ign:40 http://archive.ubuntu.com/ubuntu zesty-backports/restricted all Packages
Ign:41 http://archive.ubuntu.com/ubuntu zesty-backports/main all Packages
Ign:42 http://archive.ubuntu.com/ubuntu zesty-backports/restricted amd64 Packages
Ign:43 http://archive.ubuntu.com/ubuntu zesty-backports/multiverse amd64 Packages
Err:14 http://archive.ubuntu.com/ubuntu zesty/universe Sources
  404  Not Found [IP: 91.189.88.162 80]
Ign:16 http://archive.ubuntu.com/ubuntu zesty/main all Packages
Ign:18 http://archive.ubuntu.com/ubuntu zesty/main amd64 Packages
Ign:20 http://archive.ubuntu.com/ubuntu zesty/restricted amd64 Packages
Ign:22 http://archive.ubuntu.com/ubuntu zesty/multiverse all Packages
Ign:23 http://archive.ubuntu.com/ubuntu zesty/universe amd64 Packages
Ign:24 http://archive.ubuntu.com/ubuntu zesty/universe all Packages
Ign:25 http://archive.ubuntu.com/ubuntu zesty/restricted all Packages
Ign:26 http://archive.ubuntu.com/ubuntu zesty/multiverse amd64 Packages
Err:27 http://archive.ubuntu.com/ubuntu zesty-updates/universe Sources
  404  Not Found [IP: 91.189.88.162 80]
Ign:28 http://archive.ubuntu.com/ubuntu zesty-updates/multiverse amd64 Packages
Ign:29 http://archive.ubuntu.com/ubuntu zesty-updates/main amd64 Packages
Ign:30 http://archive.ubuntu.com/ubuntu zesty-updates/universe all Packages
Ign:31 http://archive.ubuntu.com/ubuntu zesty-updates/multiverse all Packages
Ign:32 http://archive.ubuntu.com/ubuntu zesty-updates/main all Packages
Ign:33 http://archive.ubuntu.com/ubuntu zesty-updates/restricted amd64 Packages
Ign:34 http://archive.ubuntu.com/ubuntu zesty-updates/universe amd64 Packages
Ign:35 http://archive.ubuntu.com/ubuntu zesty-updates/restricted all Packages
Err:36 http://archive.ubuntu.com/ubuntu zesty-backports/universe amd64 Packages
  404  Not Found [IP: 91.189.88.162 80]
Ign:37 http://archive.ubuntu.com/ubuntu zesty-backports/main amd64 Packages
Ign:38 http://archive.ubuntu.com/ubuntu zesty-backports/universe all Packages
Ign:39 http://archive.ubuntu.com/ubuntu zesty-backports/multiverse all Packages
Ign:40 http://archive.ubuntu.com/ubuntu zesty-backports/restricted all Packages
Ign:41 http://archive.ubuntu.com/ubuntu zesty-backports/main all Packages
Ign:42 http://archive.ubuntu.com/ubuntu zesty-backports/restricted amd64 Packages
Ign:43 http://archive.ubuntu.com/ubuntu zesty-backports/multiverse amd64 Packages
Reading package lists...
W: The repository 'http://security.ubuntu.com/ubuntu zesty-security Release' does not have a Release file.
W: The repository 'http://archive.ubuntu.com/ubuntu zesty Release' does not have a Release file.
W: The repository 'http://archive.ubuntu.com/ubuntu zesty-updates Release' does not have a Release file.
W: The repository 'http://archive.ubuntu.com/ubuntu zesty-backports Release' does not have a Release file.
E: Failed to fetch http://security.ubuntu.com/ubuntu/dists/zesty-security/universe/source/Sources  404  Not Found [IP: 91.189.88.149 80]
E: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/zesty/universe/source/Sources  404  Not Found [IP: 91.189.88.162 80]
E: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/zesty-updates/universe/source/Sources  404  Not Found [IP: 91.189.88.162 80]
E: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/zesty-backports/universe/binary-amd64/Packages  404  Not Found [IP: 91.189.88.162 80]
E: Some index files failed to download. They have been ignored, or old ones used instead.
Error executing command, exiting`

**Comment by benjohnde** [comment]
hi @dinukasal, that's because zesty retired. The repositories are not online anymore. Because of that, @schrieveslaach created a pull request https://github.com/jangrewe/gitlab-ci-android/pull/35 for that.

In the meantime, I forked this repo, merged all the pull requests and created an image at docker hub https://github.com/benjohnde/gitlab-ci-android. Additionally, I also played around with a much smaller alpine image https://github.com/benjohnde/docker-android-alpine. Maybe this helps you out.
